ilastik/applets/multicut/multicutGui.py

```diff
@@ -72,6 +72,14 @@ def stopAndCleanUp(self):
     ###########################################
     ###########################################
 
+    def __init_subclass__(cls, **kwargs):
+        """Make sure Mixin can only be used with LayerViewerGui"""
+        parent = LayerViewerGui
+        if parent not in cls.mro():
+            breakpoint()
```

Comment:
Stray `breakpoint()`?

Also, after some time I realized that the original suggestion could be simplified just to `assert issubclass(cls, LayerViewerGui)` in `__init_subclass__` (it's just a sanity check after all, users won't see that anyway).

Reply:
ah, thank you for being so thorough :D I have almost merged it ... And I agree that an assert is good enough here.